<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QWidget" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLineEdit" name="pythonExecTxt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="placeholderText">
    <string>/path/to/python_executable</string>
   </property>
  </widget>
  <widget class="QLabel" name="pythonLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This field should have the python executable name. In Mac, Linux or Unix this can be found out by typing &amp;quot;which python&amp;quot; in a terminal.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Python Executable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>75</y>
     <width>723</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>In order to work properly, MetaPathways needs to know thse paths</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>123</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbDirectoryTxt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Location of database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>26</y>
     <width>723</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="metapathwaysLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>167</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The folder that contains the functional and taxonomic reference databases. This folder is usually names as &amp;quot;MetaPathways_DBs&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Database Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pythonBrowseButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pgdbFolderTxt">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>359</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>/path/to/ptools-local/pgdbs/users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pgdbFolderBrowseButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>167</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="metapathwaysLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>209</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the location of the MetaPathways_Python directory that contains the actual pipeline stages, that is written mostly in Python&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>MetaPathways Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathMetaPathwaysTxt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>MetaPathways folder location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="metapathwaysBrowseButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>135</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>224</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="databaseButton">
   <property name="geometry">
    <rect>
     <x>623</x>
     <y>186</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>461</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Once you are done filling in the paths, please click the proceed button to update the paths</string>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>404</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pathologicButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>87</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathologicTxt">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>423</y>
     <width>321</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>pathway tool executable</string>
   </property>
  </widget>
  <widget class="QLabel" name="metapathwaysLabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>423</y>
     <width>225</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location of the PathwayTools executable that is used to view existing  ePGDBs and also create new ePGDBs. Usually it is installed in the folder structure ~/pathway-tools/pathway-tools  under your home directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Pathway Tools Executable</string>
   </property>
  </widget>
  <widget class="QLabel" name="pgdbFolderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>359</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location of the pgdb folder that stores the ePGDBs. Usually it is installed in the folder structure ~/ptools-local/pgdbs/user under your home directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>PGDB Folder Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="executablesLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>241</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is the location of the folder that has the OS compatible binaries, such as, lastal, blastp, etc under the &amp;lt;executables&amp;gt; folder in MetaPathways Directory. The options in the drop down menu become available after the &amp;quot;MetaPathways Directory&amp;quot; is specified. In order to support a new flavour of OS the user must create a folder under the &amp;lt;executables&amp;gt; directory and compile the binaries from the sources folder in that OS and put the respective binaries in the folder.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Dir for OS specific executables</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>277</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="executablesCombo">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>240</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Validate paths/executables</string>
   </property>
  </widget>
  <widget class="QLabel" name="check3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="check4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="check0">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>140</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="check2">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>242</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="check1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="check5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>421</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/check.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="star1">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>137</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Required field&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/requiredstar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="star2">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>190</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Required field</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources.qrc">:/images/requiredstar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pythonExecTxt</tabstop>
  <tabstop>pythonBrowseButton</tabstop>
  <tabstop>pgdbFolderTxt</tabstop>
  <tabstop>pgdbFolderBrowseButton</tabstop>
  <tabstop>pathMetaPathwaysTxt</tabstop>
  <tabstop>metapathwaysBrowseButton</tabstop>
  <tabstop>dbDirectoryTxt</tabstop>
  <tabstop>databaseButton</tabstop>
  <tabstop>pathologicTxt</tabstop>
  <tabstop>pathologicButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
